api/room_handler_test.go

```diff
@@ -1585,6 +1603,7 @@ forwarders:
 				Return(redis.NewStringStringMapResult(map[string]string{
 					"status":   "ready",
 					"lastPing": "1632405900",
+					"metadata": `{"region":"us","type":"sometype"}`,
```

Comment:
We added the metadata in all mocks, but we had a problem in the past during the serialization of the data into API. Is it possible to check if the metadata will be "correct" after we call the API?

Also, WDYT about add a scenario where metadata is just en empty {} (more commons scenario in prod)